Makefile

```diff
@@ -0,0 +1,65 @@
+.PHONY: all clone-quickstart install-quickstart start-quickstart install-openmct-yamcs sanity-test build-example test-getopensource test-e2e clean
+
+test-all: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example test-e2e
+
+start-all: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example start-openmct
+
+clone-quickstart:
+	@echo "Running target: clone-quickstart"
+	@echo "Current working directory: $(shell pwd)"
+	if [ ! -d "quickstart" ]; then \
+		git clone https://github.com/yamcs/quickstart; \
+	else \
+		echo "Directory 'quickstart' already exists."; \
+	fi
+
+install-quickstart:
+	@echo "Running target: install-quickstart"
+	@cd quickstart/docker && $(MAKE) wait-for-sent
+
+start-quickstart:
+	@echo "Running target: start-quickstart"
+	@cd quickstart/docker && $(MAKE) all
+
+reset-quickstart:
+	@echo "Running target: reset-quickstart"
+	@cd quickstart/docker && $(MAKE) yamcs-simulator-restart
+
+install-openmct-yamcs:
+	@echo "Running target: install-openmct-yamcs"
+	npm install || exit 1
```

Comment:
are these `exit 1` doing anything? if it fails, won't the Makefile fail anyway?

Reply:
@scottbell that's correct. I'll remove them